django/db/models/sql/where.py

```diff
@@ -41,7 +44,7 @@ def split_having(self, negated=False):
         # If the effective connector is OR and this node contains an aggregate,
         # then we need to push the whole branch to HAVING clause.
         may_need_split = (
-            (in_negated and self.connector == AND) or
+            (in_negated and self.connector == (AND, XOR)) or
```

Comment:
Flagging this because I think the change was made based on Mariusz' suggestion (somewhere upthread), but the operator has accidentally been left unchanged. This should be `in` rather than `==` I think?

Reply:
Oof, ouch. Thanks for that. Fixed